static/src/deploys-radiator/app/main.js

```diff
@@ -34,14 +34,39 @@ const run = () => {
         'rss',
         'sport'
     ]);
-    const filterWhitelisted = (deploys) => deploys
+    const unifiedWhitelist = List([
+        'all'
+    ]);
+    const filterServerWhitelisted = (deploys) => deploys
         .filter(deploy => serverWhitelist.contains(deploy.projectName))
         .toList();
+    const filterUnifiedWhitelisted = (deploys) => deploys
+        .filter(deploy => unifiedWhitelist.contains(deploy.projectName))
+        .toList();
+    const mostRecentOf = (serverDeploys, unifiedDeploys) => {
+        if (serverDeploys.isEmpty()) return unifiedDeploys;
+        if (unifiedDeploys.isEmpty()) return serverDeploys;
+        const serverTime = serverDeploys.sort((a, b) => a.time.getTime() - b.time.getTime()).last().time.getTime();
+        const unifiedTime = unifiedDeploys.sort((a, b) => a.time.getTime() - b.time.getTime()).last().time.getTime();
```

Comment:
Not quite, but I've tried using it and it seems to work as expected so the difference is possibly not important.